pkg/kubecost/allocation.go

```diff
@@ -662,6 +655,7 @@ type AllocationSet struct {
 	allocations  map[string]*Allocation
 	externalKeys map[string]bool
 	idleKeys     map[string]bool
+	FromSource   string
```

Comment:
I think commenting this field would help.

Reply:
Done